app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -55,6 +57,8 @@ import kotlin.math.max
 class TabsTrayFragment : AppCompatDialogFragment() {
 
     private var fabView: View? = null
+    private var _currentOrientation = MutableStateFlow(Configuration.ORIENTATION_UNDEFINED)
+    @VisibleForTesting internal val currentOrientation: StateFlow<Int> = _currentOrientation
```

Comment:
We already have a way to store state in the fragment (TabsTrayStore/TabsTrayState), let's move this orientation state use the existing concept instead of introducing a place for it.